### ISSUE TITLE
False positive source compatibility problem

### ISSUE THREAD

**Issue by lbergelson** [lvc/japi-compliance-checker#51]
I noticed an issue where it claims something isn't source compatible when it actually is.

```
public int getValue() {...}  

public int getValue(Object... objects){ ... }
```

This change is binary incompatible, but it should be source compatible since you can still invoke it with no arguments.
